root/static/scripts/edit/URLCleanup.js

```diff
@@ -1747,6 +1747,28 @@ const CLEANUPS = {
       return /^https:\/\/myspace\.com\//.test(url);
     },
   },
+  'napster': {
+    match: [new RegExp('^(https?://)?([a-z]{2}\\.)?napster\\.com', 'i')],
```

Comment:
It doesn’t handle `ẁww` subdomain. Many other legitimate (old) URLs can be found in the MBDB (see <https://gist.github.com/yvanzo/ef2b29691be4a3ec4c4d67dffc3a1240>) and probably elsewhere.

Reply:
I made it handle www. and app.

Other older URLs seem to redirect so we can hand-fix the few wrong ones. I did that for a few.

Reply:
Maybe `vivomusica.` should be handled too as it seem to be still alive?
(I don’t know its purpose though.)

Reply:
I saw that, but that's supposed to be "VivoMusica by Napster", so not exactly Napster? It's confusing.

Reply:
Ok, please document it in commit message then.
It seems to be rare enough to not create a specific rule for this one.

Reply:
Done. Mentioned the app and www redirect on the commit message too.